torchtext/data/datasets_utils.py

```diff
@@ -33,6 +34,25 @@ def _clean_xml_file(f_xml):
                 fd_txt.write(e.text.strip() + '\n')
 
 
+def _clean_inner_xml_file(outfile, stream):
+    """Accepts an output filename and a stream of the byte contents of an XML file
+    within a tarball and writes the cleaned contents to a new, untarred file.
```

Comment:
I don't think it's necessary to mention `within a tarball and writes the cleaned contents to a new, untarred file.`. It could come from anywhere :)